flexget/plugins/filter/upgrade.py

```diff
@@ -68,31 +68,54 @@ def prepare_config(self, config):
         config.setdefault('propers', True)
         return config
 
-    def filter_entries(self, entries, existing, target, action_on_lower):
+    def filter_entries(self, entries, existing, target, propers, action_on_lower):
 
         target_requirement = qualities.Requirements(target) if target else None
+        target_quality = qualities.Quality(target) if target else None
```

Comment:
Hmm. Is 'target' a specific quality, or a quality requirement? It doesn't really make sense for it to be both.